app/views/catalog/_svccat_tree_show.html.haml

```diff
@@ -44,10 +44,12 @@
     .form-group
       .col-md-1{:align => "center"}
         #buttons
+          - template_invalid = Array(@flash_array).any? { |f| f[:level] == :warning }
```

Comment:
This is probably not how we want to do it, you have access to the `@record`, use that instead.

Reply:
Yes, I know. I was about to change it but you were faster